packages/publisher-dapp/src/Pages/Overview/index.js

```diff
@@ -1,11 +1,173 @@
+	                <CheckIcon className={classes.checkCircleIcon} />
+	              </ListItemIcon>
+	            	<ListItemText primary="AI services dashboard" />
+	            </ListItem>
+	            <ListItem>
+	            	<ListItemIcon>
+	                <CheckIcon className={classes.checkCircleIcon} />
+	              </ListItemIcon>
+	            	<ListItemText primary="Reminders and notifications" />
+	           	</ListItem>
+	           	<ListItem>
+	           		<ListItemIcon>
+	                <CheckIcon className={classes.checkCircleIcon} />
+	              </ListItemIcon>
+	            	<ListItemText primary="Transfer AGI Tokens from your perferred wallet anytime." />
+	            </ListItem>
+	            <ListItem>
+	            	<ListItemIcon>
+	                <CheckIcon className={classes.checkCircleIcon} />
+	              </ListItemIcon>
+	            	<ListItemText primary="Integrate AI services using your perferred language such as Python, Java, C++, and many more" />
+	            </ListItem>
+						</div>
+						<div className={classes.rightSideList}>
+	            <ListItem>
+	            	<ListItemIcon>
+	                <CheckIcon className={classes.checkCircleIcon} />
+	              </ListItemIcon>
+	            	<ListItemText primary="Integrate AI services using your perferred language such as Python, Java, C++, and many more" />
+	            </ListItem>
+	            <ListItem>
+	            	<ListItemIcon>
+	                <CheckIcon className={classes.checkCircleIcon} />
+	              </ListItemIcon>
+	            	<ListItemText primary="Reminders and notifications" />
+	           	</ListItem>
+	           	<ListItem>
+	           		<ListItemIcon>
+	                <CheckIcon className={classes.checkCircleIcon} />
+	              </ListItemIcon>
+	            	<ListItemText primary="Transfer AGI Tokens from your perferred wallet anytime." />
+	            </ListItem>
+	            <ListItem>
+	            	<ListItemIcon>
+	                <CheckIcon className={classes.checkCircleIcon} />
+	              </ListItemIcon>
+	            	<ListItemText primary="AI services dashboard" />
+	            </ListItem>
+						</div>
+	        </List>
+	        <StyledButton btnText="start your enroll" btnType="blue" />
+	      </div>
+      </Grid>
+      <Grid item xs={12} sm={12} md={12} lg={12} className={classes.getInTouch}>
+      	<Typography variant="h2">More Questions? Get in Touch.</Typography>
+        <Typography variant="body1">Lorem ipsum dolor sit amet, ut sea homero forensibus. Ea veri indoctum nam, nec ea nulla concludaturque, graeco assentior at nam. Fugit veritus propriae sed at, in usu labores offendit. Pri veniam vivendum in, elitr latine sed te. His simul inimicus negle ge ntur ex, an vix praesent iracundia.</Typography>
+      	<form>
+	        <TextField
+	          id="outlined-number"
+	          label="Email"
+	          type="text"
+	          className={classes.textField}
```

Comment:
there is no textField class in styles.js. remove it

Reply:
check if any other classNames is not there in the styles.js. remove them as well.